src/string_buffer.rs

```diff
@@ -198,6 +204,41 @@ impl fmt::Display for StringBuffer {
     }
 }
 
+impl fmt::Debug for StringBuffer {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "\"{}\"", self)
```

Comment:
The debug impl should reflect the actual structure of the StringBuffer, i.e., show each node.